src/components/ModalAddManyTokens.js

```diff
@@ -8,6 +8,7 @@
 import React from 'react';
 import $ from 'jquery';
 import tokens from '../utils/tokens';
```

Comment:
Missing import?

Reply:
Actually not, I have some utils methods that come from the `hathorLib` and others that come from the wallet utils itself. I have a `tokens.js` file in the wallet utils. Like the other comment, do you think we should rename it?